api/models.py

```diff
@@ -105,8 +105,11 @@ def to_dict(self):
             'twitter_username_verified': self.twitter_username_verified,
             'twitter_username_verification_tweet': f"https://twitter.com/{app.config['TWITTER_USER']}/status/{self.twitter_username_verification_tweet_id}",
             'contribution_percent': self.contribution_percent,
-            'has_auctions': len(self.auctions) > 0,
-            'has_bids': len(self.bids) > 0}
+            'has_auctions': len(self.auctions.all()) > 0,
+            'has_bids': len(self.bids) > 0,
+            'active_auction_count': len(self.auctions.filter(Auction.end_date >= now).all()),
+            'past_auction_count': len(self.auctions.filter(Auction.end_date <= now).all()),
```

Comment:
So here we are essentially doing 2 separate SELECTs on the database, only to get the two counts, plus we do one more SELECT after that to load the actual auctions (in the other endpoint)? Can't we just load the auctions and get the count from there?